content/rule_book/material_and_questions.md

```diff
@@ -92,7 +92,7 @@ Question Type maximums can be exceeded in Overtime.
 - Used to distinguish exact same words or phrases from the material.
 - The entire reference question is part of the required question + answer.
 - If a quizzer needs to provide the reference question, the provided reference question must:
-    - Be of the exact same type as the one on the card (CR, CVR, MACR, MACVR). The quizmaster and/or answer judge must check to confirm this.
+    - Be of the exact same type (CR, CVR, MACR, MACVR) as the written question.
```

Comment:
I changed every reference to "on the card" to "the written question"

Reply:
Agreed. The concept of "card" in Quizzing is deprecated.

Reply:
It's NUTS how iterating continues to catch small improvements that can still be FE.

Reply:
"How to get really good at anything:

1. Do it badly.
2. Iterate."